docs/pages/dropdown.js

```diff
@@ -551,14 +551,14 @@ function TruncationDropdownExample() {
         name="Dropdown.Item"
         id="Dropdown.Item"
         generatedDocGen={generatedDocGen.DropdownItem}
-        excludeProps={['index']}
+        excludeProps={['_index']}
```

Comment:
You shouldn't need to manually exclude these — we filter out any props with an underscore prefix

Reply:
DONE